frontend/scripts/svgr.js

```diff
@@ -0,0 +1,99 @@
+const fs = require('fs')
+const path = require('path')
+const svgr = require('@svgr/core').default
+const prompts = require('prompts')
+// https://gist.github.com/pladaria/69321af86ce165c2c1fc1c718b098dd0
+const svgoConfig = {
+  plugins: [{ removeViewBox: false }],
+}
```

Comment:
Did you also want to add the `{ "cleanupIDs": false }` that was in the original `svgo.json` config?

Reply:
When I was testing it, enabling `cleanupIDs` seemed to help clean up the SVGs but not affect how the SVG was rendered. Was the original value in `svgo.json` preventing a bug or something that I maybe missed?

Reply:
Merging for now, but will follow up with a quick PR if `cleanupIDs` is needed

Reply:
Yeah the whole point of the settings I added to the `svgo.json` was to help clean up and optimize the SVG output, in part so that they are easier to work with, especially if in some cases there are animations or all kinda of things we might want to manipulate in the SVG. In general I think it's a good idea to check with each other in a case like this before removing something one of us put there for a reason, but it's not a huge deal of course.

Reply:
We might be saying the same thing, but I removed `cleanupIDs: false`, so that that the default of true would be enabled, meaning that the SVG would be cleaned up even more. I checked the the previous SVGs to make sure I didn't break anything.